internal/connect/utils.go

```diff
@@ -117,10 +134,24 @@ func uploadFile(url, filePath string, formFields map[string]any) error {
 	client := &http.Client{
 		Timeout: 20 * time.Minute,
 	}
-	_, err = sling.New().Client(client).Base(url).Set("Content-Type", writer.FormDataContentType()).Post("").Body(&buffer).ReceiveSuccess(nil)
+
+	// Create the HTTP request
+	request, err := http.NewRequest("POST", url, &buffer)
 	if err != nil {
 		return err
 	}
+	// Set the Content-Type header to multipart/form-data
+	request.Header.Set("Content-Type", writer.FormDataContentType())
+	response, err := client.Do(request)
+	if err != nil {
+		return err
+	}
+	defer response.Body.Close()
+
+	if response.StatusCode >= 400 {
+		responseBody, err := ioutil.ReadAll(response.Body)
+		return fmt.Errorf("[Response] %s [Error] %v", string(responseBody), err)
```

Comment:
I would assume that if there were an error, `responseBody` would be `""` and this wrapped error would look poorly formatted.

Reply:
if there is error in aws api will be like
responseBody : <?xml version="1.0" encoding="UTF-8"?>
<Error><Code>EntityTooSmall</Code><Message>Your proposed upload is smaller than the minimum allowed size</Message><ProposedSize>0</ProposedSize><MinSizeAllowed>1024</MinSizeAllowed><RequestId>JV8AQKW05T8KX45N</RequestId><HostId>AaKn9Vv2pqnC4T5pazA+TaJHFGSTxO5ETLRIO2IEimUWu97kZsStTLcq9jOoSk3vHIPQqXJ+J4k=</HostId></Error>

Reply:
Oops, I meant if `err != nil`, then `responseBody` should be `""`.